packages/marketing-modal/src/MarketingModal.tsx

```diff
@@ -152,6 +161,8 @@ interface MarketingModalProps {
   linkText: string;
   darkMode?: boolean;
   closeIconColor?: CloseIconColor;
```

Comment:
Would be cool if we could programatically set this by checking the color of what's underneath it. Super out-of-scope though 😆

Reply:
yeah that would be cool